file-check/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: file-check
```

Comment:
Please change the directory name and other places as well.

```suggestion
name: check-file-existence
```

Reply:
Changed b13e007

Reply:
Thank you!